bibtexparser/bibdatabase.py

```diff
@@ -103,6 +116,48 @@ def expand_string(self, name):
         except KeyError:
             raise(UndefinedString(name))
 
+    def _add_missing_from_crossref_entry(self, entry, dependencies=set()):
+        if entry['ID'] in self._crossref_updated:
+            return
+
+        if entry['_crossref'] not in self._entries_dict:
+            logger.error("Crossref reference %s for %s is missing.",
+                         entry['_crossref'],
+                         entry['ID'])
+            return
```

Comment:
Don't we want to fail here?

Reply:
I don't think it makes sense to fail here, because biber/biblatex etc. are also not failing but logging an undefined reference instead, and consistency with them is probably what users expect the behavior to be.

Reply:
Marking as resolved, so that GitHub updates PR properly.